core/src/test/resources/schema/postgres/partitioned-schema.sql

```diff
@@ -42,7 +42,6 @@ CREATE SEQUENCE journal_ordering_seq OWNED BY public.journal.ordering;
 
 CREATE EXTENSION IF NOT EXISTS intarray WITH SCHEMA public;
 CREATE INDEX journal_tags_idx ON public.journal USING GIN (tags public.gin__int_ops);
-CREATE INDEX journal_persistence_sequence_idx ON public.journal USING BTREE (persistence_id, sequence_number);
```

Comment:
This SQL should contain the effective schema for the current revision of the plugin, so that applying this script on a clean database should result in the complete schema without bothering about Flyways, running additional SQL scripts etc.

IMO you should trigger procedure for creating concurrent indexes here. Otherwise you will miss one of the essential indexes.

Reply:
Please ignore above comment. I forgot that in case when we start with a fresh database, plugin will create a new schema with all the necessary indices.